boxoffice/assets/js/views/org.js

```diff
@@ -2,30 +2,89 @@
 var Ractive = require('ractive');
 import {eventBus} from './main_admin.js'
 var NProgress = require('nprogress');
-import {fetch, urlFor, setPageTitle} from '../models/util.js';
+import {Util, fetch, post, getFormParameters, urlFor, setPageTitle} from '../models/util.js';
 import {orgTemplate} from '../templates/org.html.js';
 import {SideBarView} from './sidebar.js'
 
 export const OrgView = {
   render: function({org_name}={}) {
+    const DEFAULT = {
+      showForm: true,
+      hideForm: false,
+      btnDisable: true,
+      btnEnable: false,
+      empty: ""
+    };
+
     fetch({
       url: urlFor('view', {resource: 'o', id: org_name, root: true})
-    }).then(function({id, org_title, item_collections}){
+    }).then(function({id, org_title, item_collections, form}) {
+      let AddICFormComponent = Ractive.extend({
+        isolated: false,
+        template: Util.getFormTemplate(form),
+        data: {
+          addFormId: Util.getElementId(form),
```

Comment:
`addFormId` seems like a name for a function, but here it's just a variable. `formId` perhaps?

Reply:
Fixed